app/views/notes/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+  <%= form_with model: [ current_user, current_user.notes.build ] do |form| %>
+  <%# .........................template........................... %>
+    <div id="algin-form" class="formbg">
+      <div class="form-group">
+        <h5>Add a Note</h5>
+        <label for="note" class="text-white">Title</label>
+        <%= form.text_field :title, class: 'form-control', cols: "30", rows: "5" %>
```

Comment:
text_field doesn’t need cols and rows, remove it please.

Reply:
done